fastseq_cli/transformers_generate.py

```diff
@@ -17,6 +18,37 @@ def chunks(lst, n):
     for i in range(0, len(lst), n):
         yield lst[i:i + n]
 
+def sort_sentences(input_ids: List[str], reverse: bool=False):
```

Comment:
replace 'reverse' by 'descending' is more clear.

Reply:
`reverse` is used by python built-in [sorted(...)](https://docs.python.org/3/library/functions.html#sorted) function. Will it be better to use the same name here?

Reply:
sounds good